src/table.rs

```diff
@@ -38,7 +37,7 @@ impl ToString for ColumnType {
 }
 
 #[derive(Clone, Debug, Hash, PartialEq, Eq)]
-enum Datum {
+pub enum Datum {
     Int(i64),
     String(String),
```

Comment:
Maybe `varchar(n)` and `char(n)` can be tested respectively

Reply:
We can. I don't quite understand why it is needed. Is there a big difference between them?